zeppelin-server/src/main/java/org/apache/zeppelin/realm/kerberos/KerberosRealm.java

```diff
@@ -91,6 +94,9 @@
   private boolean isCookiePersistent = false;
   private String signatureSecretFile = null;
   private String signatureSecretProvider = "file";
+  private String logout = "/logout";
+  private String providerUrl = "https://domain.example.com/";
+  private String redirectParam = "originalUrl";
```

Comment:
I think we will need `logoutAPI` as well, similar to the implementation of KnoxJwtRealm.